tests/server/server-executeBulkWrite-005.phpt

```diff
@@ -24,7 +24,10 @@ foreach ($writeConcerns as $wc) {
     var_dump($result->getInsertedCount());
 }
 
-$command = new MongoDB\Driver\Command(array('drop' => COLLECTION_NAME));
+$command = new MongoDB\Driver\Command([
+    'delete' => COLLECTION_NAME,
+    'deletes' => [ [ 'q' => (object) [], 'limit' => 0 ] ]
+]);
```

Comment:
Is this to avoid issue with not having permissions to `drop` (althouga h writes are allowed)?

Rather than hard-code delete commands like this, I think it'd be clearer to use `executeBulkWrite()`.

Reply:
"althouga h" ← **:-þ**

Yes, that is the reason. I'll change it to use ``executeBulkWrite()``.